README.md

```diff
@@ -73,7 +76,7 @@ Prior to installing oci-systemd-hook, install the following packages on your lin
 In Fedora, you can use this command:
 
 ```
- dnf -y install \
+ yum -y install \
```

Comment:
At line 76 we say "In Fedora, ..." should that be changed to "In Fedora or CentOS"?